src/main.cpp

```diff
@@ -28,5 +29,7 @@ int main(int argc, char **argv)
     setenv("XDG_CACHE_HOME", appCachePath.toUtf8().constData(), 1);
 
     LunaQmlApplication application(argc, argv);
+
+    QtWebEngine::initialize();
```

Comment:
Unfortunately, and for a reason I ignore, calling initialize is mandatory before using QtWebEngine.
However, there are two mistakes in this PR:
 - there is an #include missing for <QtWebEngine/qtwebengineglobal.h>
 - the recipe will have to depend on QtWebEngine, and it is a bit disgraceful. Also for other QML apps like Phone it may induce some overhead.
I'll try to propose a change in a browser's plugin equivalent to this one, but specific to the browser QML app.